types/stripe-js/checkout.d.ts

```diff
@@ -423,9 +423,7 @@ export type StripeCheckoutConfirmResult =
   | {type: 'success'; success: StripeCheckoutSession}
   | {type: 'error'; error: ConfirmError};
 
-type RunServerUpdateFunction = (
-  session: StripeCheckoutSession
-) => Promise<void>;
+type RunServerUpdateFunction = () => Promise<any>;
```

Comment:
use `any` since we don't have any requirements of the resolved value.

Reply:
(optional) `unknown` may be more accurate here as the universal supertype

Reply:
Fixed https://github.com/stripe/stripe-js/pull/712/commits/aa01b7f4662923200055d24737b9ea34f91d5284

Thank you.